spec/test-outputs/www-integration.out.vcl

```diff
@@ -382,6 +386,12 @@ sub vcl_fetch {
     return (pass);
   }
 
+  # Pass through good responses to POST requests (i.e. cache responses to POST
+  # requests that are the result of client error).
+  if (req.request == "POST" && !(beresp.status >= 400 && beresp.status <= 499)) {
```

Comment:
The numbers here seem potentially too broad. It feels like we could legitimately be returning a 400, 403 or a 422 to a user based on their request that we wouldn't want to cache. 

Off the top of my head I think it's only 404 and 405 we'd want to cache

Reply:
That sounds reasonable. I limited it to client errors initially (we don't want to cache things for a server error, for example), but starting with a smaller subset until necessary it's probably sensible.